PHP/PHPhomework1.php

```diff
@@ -0,0 +1,11 @@
+<?php
+  
+$x = 0;
+for ($i = 0; $i < 51; $i++) {
+    if ($i % 2 === 1) {
```

Comment:
@toru828 can you try to calculate the sum of odd numbers without the condition ?
example: `$x += $i % 2 * $i;`